src/01-general/users.apib

```diff
@@ -114,6 +124,11 @@ Get details for a single user.
             + first_name: `John` (string)
             + last_name: `Smith` (string)
             + email: `john@teamleader.eu` (string)
+            + emails (array)
+                + (object)
+                    + email: `john@teamleader.eu` (string)
```

Comment:
Would this replace the email? Should we state which one is the primary one?

Reply:
It replaces `email` yes but we can't remove it because that would be a breaking change

> Should we state which one is the primary one?

The one in `email` is the primary one. I can also add it in the emails object but I didn't because it didn't seem super useful to me